presidio-anonymizer/presidio_anonymizer/entities/anonymizer_config.py

```diff
@@ -0,0 +1,54 @@
+"""Handle the anonymizers data - anonymizer class and params."""
+import logging
+
+from presidio_anonymizer.anonymizers import Anonymizer
+from presidio_anonymizer.entities import InvalidParamException
+
+
+class AnonymizerConfig:
+    """Handle the anonymizers data - anonymizer class and params."""
+
+    @classmethod
+    def from_json(cls, params: dict):
+        """
+        Create AnonymizerConfig from json.
+
+        :param params: json with struct of {
```

Comment:
Maybe mention that this is an example param but would change for different anonymizers?

Reply:
Done.